File: Swift/reverse-words-in-a-string-iii.swift
class Solution {
    func reverseWords(_ s: String) -> String {
        let words = s.split(separator: " ")
        var res = ""

        for (i, word) in words.enumerated() {
            res += String(word.reversed())
            if i != words.count - 1 {
                res += " "
            }
        }

        return res
    }
}

File: Python/reverse-words-in-a-string-iii.py
class Solution:
    def reverseWords(self, s):
        """
        :type s: str
        :rtype: str
        """
        words = s.split(' ')
        res = ''

        for i, word in enumerate(words):
            res += word[::-1]
            if i != len(words) - 1:
                res += ' '

        return res
